src/main/resources/assets/paths/datasetV3.yaml

```diff
@@ -41,3 +41,41 @@ post:
       description: Dataset Name given is already in use by another dataset
     500:
       description: Internal Error (something went wrong processing a valid input)
+
+put:
+  summary: Updates a dataset from a valid dataset registration JsonSchema object
+  description: Updates a dataset from a valid dataset registration JsonSchema object
+  requestBody:
+    description: to do
+    content:
+      multipart/form-data:
+        schema:
+          type: object
+          properties:
+            dataset:
+              description: Required dataset registration json schema instance
+              type: object
+              $ref: '../schemas/DatasetRegistrationSchemaV1.yaml'
+            alternativeDataSharingPlan:
+              description: Optional alternative data sharing plan
+              type: string
+              format: binary
+            consentGroups[0].nihInstitutionalCertificationFile:
+              description: Certification file for first consent group (others omitted because swagger does not allow variable number of file uploads)
+              type: string
+              format: binary
```

Comment:
This is why I find the schema approach over-complicated. We're forced to replicate the multiple consent group approach when we really don't have to.